frontend/src/modules/tag/components/tag-popover.vue

```diff
@@ -72,41 +62,71 @@ export default {
     },
     computedVisible: {
       get() {
-        return this.modelValue !== null;
+        return this.modelValue;
       },
       set() {
-        this.$emit('update:modelValue', null);
+        this.$emit('update:modelValue', false);
       },
     },
   },
 
   watch: {
     modelValue: {
-      immediate: true,
-      handler(member) {
-        if (member) {
-          this.form = member.tags;
+      async handler(newValue) {
+        if (newValue) {
+          await this.prepareBulkUpdateTags();
         }
       },
     },
   },
 
   methods: {
     ...mapActions({
-      doUpdate: 'member/doUpdate',
+      doBulkUpdateMembersTags:
+        'member/doBulkUpdateMembersTags',
     }),
+
+    prepareBulkUpdateTags() {
+      this.bulkEditTagsModel = selectedMembers.value.reduce(
```

Comment:
Currently since you are only using `selectedMembers` to parse the request payload, this is not working for the scenario where you click "Edit tags" in one member row.
My suggestion would be to have an optional prop where you would pass a member object or array (only used when you click Edit tags for one row). -> `member`
Then we could have a computed property that saved the members we want to update and its value would be something like: `member.length ? member : selectedMembers`

Then instead of using `selectedMembers` you would use this computed property. What do you say? Feel free to choose any other approach

Reply:
+1 good idea, I would go with what you suggested.